services/crypto_get_account_balance.proto

```diff
@@ -31,55 +41,81 @@ import "query_header.proto";
 import "response_header.proto";
 
 /**
- * Get the balance of a cryptocurrency account. This returns only the balance, so it is a smaller
- * reply than CryptoGetInfo, which returns the balance plus additional information.
+ * Query to read the HBAR balance of an account or contract.
+ *
+ * This query SHALL return _only_ the HBAR balance for an account
+ * or smart contract. Early releases of the network would return all
+ * fungible/common token balances, but HIP-367 made it infeasible to
+ * return all such balances. This query SHALL NOT return any information
+ * beyond the current HBAR balance.
```

Comment:
We allow aliases in the account in addition to just account number for getting accountInfo